Website build on CI is down due to Julia error

## Description

This is in the error log.

```
julia --color=yes --project=./ ./make.jl
┌ Warning: The environment variable `MXNET_HOME` has been renamed, please use `MXNET_ROOT` instead.
└ @ MXNet.mx /work/mxnet/docs/build_version_doc/apache-mxnet/master/julia/src/base.jl:63
ERROR: LoadError: LoadError: Cannot find or load libmxnet.so. Please see the document on how to build it.
```

Seems to be failing again due to a different reason: http://jenkins.mxnet-ci.amazon-ml.com/blue/organizations/jenkins/mxnet-validation%2Fwebsite/detail/PR-17302/13/pipeline/
```
[2020-02-07T01:31:11.534Z] Traceback (most recent call last):

[2020-02-07T01:31:11.534Z]   File "/home/jenkins_slave/.local/bin/mkdocs", line 8, in <module>

[2020-02-07T01:31:11.534Z]     sys.exit(cli())

[2020-02-07T01:31:11.534Z]   File "/home/jenkins_slave/.local/lib/python3.5/site-packages/click/core.py", line 764, in __call__

[2020-02-07T01:31:11.534Z]     return self.main(*args, **kwargs)

[2020-02-07T01:31:11.534Z]   File "/home/jenkins_slave/.local/lib/python3.5/site-packages/click/core.py", line 717, in main

[2020-02-07T01:31:11.534Z]     rv = self.invoke(ctx)

[2020-02-07T01:31:11.534Z]   File "/home/jenkins_slave/.local/lib/python3.5/site-packages/click/core.py", line 1137, in invoke

[2020-02-07T01:31:11.534Z]     return _process_result(sub_ctx.command.invoke(sub_ctx))

[2020-02-07T01:31:11.534Z]   File "/home/jenkins_slave/.local/lib/python3.5/site-packages/click/core.py", line 956, in invoke

[2020-02-07T01:31:11.534Z]     return ctx.invoke(self.callback, **ctx.params)

[2020-02-07T01:31:11.534Z]   File "/home/jenkins_slave/.local/lib/python3.5/site-packages/click/core.py", line 555, in invoke

[2020-02-07T01:31:11.534Z]     return callback(*args, **kwargs)

[2020-02-07T01:31:11.534Z]   File "/home/jenkins_slave/.local/lib/python3.5/site-packages/mkdocs/__main__.py", line 162, in build_command

[2020-02-07T01:31:11.534Z]     site_dir=site_dir

[2020-02-07T01:31:11.534Z]   File "/home/jenkins_slave/.local/lib/python3.5/site-packages/mkdocs/config/base.py", line 197, in load_config

[2020-02-07T01:31:11.534Z]     errors, warnings = cfg.validate()

[2020-02-07T01:31:11.534Z]   File "/home/jenkins_slave/.local/lib/python3.5/site-packages/mkdocs/config/base.py", line 115, in validate

[2020-02-07T01:31:11.534Z]     post_failed, post_warnings = self._post_validate()

[2020-02-07T01:31:11.534Z]   File "/home/jenkins_slave/.local/lib/python3.5/site-packages/mkdocs/config/base.py", line 95, in _post_validate

[2020-02-07T01:31:11.534Z]     config_option.post_validation(self, key_name=key)

[2020-02-07T01:31:11.534Z]   File "/home/jenkins_slave/.local/lib/python3.5/site-packages/mkdocs/config/config_options.py", line 432, in post_validation

[2020-02-07T01:31:11.534Z]     config[key_name] = theme.Theme(**theme_config)

[2020-02-07T01:31:11.534Z]   File "/home/jenkins_slave/.local/lib/python3.5/site-packages/mkdocs/theme.py", line 48, in __init__

[2020-02-07T01:31:11.534Z]     self._load_theme_config(name)

[2020-02-07T01:31:11.534Z]   File "/home/jenkins_slave/.local/lib/python3.5/site-packages/mkdocs/theme.py", line 78, in _load_theme_config

[2020-02-07T01:31:11.534Z]     theme_dir = utils.get_theme_dir(name)

[2020-02-07T01:31:11.534Z]   File "/home/jenkins_slave/.local/lib/python3.5/site-packages/mkdocs/utils/__init__.py", line 278, in get_theme_dir

[2020-02-07T01:31:11.534Z]     return os.path.dirname(os.path.abspath(theme.load().__file__))

[2020-02-07T01:31:11.534Z]   File "/usr/local/lib/python3.5/dist-packages/pkg_resources/__init__.py", line 2443, in load

[2020-02-07T01:31:11.534Z]     self.require(*args, **kwargs)

[2020-02-07T01:31:11.534Z]   File "/usr/local/lib/python3.5/dist-packages/pkg_resources/__init__.py", line 2466, in require

[2020-02-07T01:31:11.534Z]     items = working_set.resolve(reqs, env, installer, extras=self.extras)

[2020-02-07T01:31:11.534Z]   File "/usr/local/lib/python3.5/dist-packages/pkg_resources/__init__.py", line 792, in resolve

[2020-02-07T01:31:11.534Z]     raise VersionConflict(dist, req).with_context(dependent_req)

[2020-02-07T01:31:11.534Z] pkg_resources.VersionConflict: (Markdown 3.2 (/home/jenkins_slave/.local/lib/python3.5/site-packages), Requirement.parse('markdown<3.2'))

[2020-02-07T01:31:11.534Z] Makefile:19: recipe for target 'all' failed
```

--- a/julia/docs/Makefile
+++ b/julia/docs/Makefile
@@ -20,7 +20,12 @@ all:
 	  'using Pkg; \
 	   Pkg.develop(PackageSpec(name="MXNet", path = joinpath(pwd(), "..")))'
 	julia --color=yes --project=./ ./make.jl
-	pip install --user Markdown==3.1 pygments mkdocs mkdocs-material python-markdown-math
+	pip install --user Markdown==3.1 \
+          mkdocs==1.0.4 \
+	  mkdocs-material==4.6.0 \
+	  pygments==2.5.2 \
+	  pymdown-extensions==6.2.1 \
+	  python-markdown-math==0.6
 	export LC_ALL="C.UTF-8"
 	export LANG="C.UTF-8"
 	echo $(LC_ALL)

